packages/tatum-btc/src/wallet/wallet.ts

```diff
@@ -22,11 +22,11 @@ export const generateBtcWallet = async (testnet: boolean, mnem: string): Promise
 
 /**
  * Generate wallet
- * @param options.testnet testnet or mainnet version of address
- * @param options.mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param options.testnet optional testnet or mainnet version of address
  * @returns wallet or a combination of address and private key
  */
-export const generateWallet = (options: { testnet?: boolean; mnemonic?: string }) => {
-  const mnem = options?.mnemonic ? options.mnemonic : generateMnemonic(256)
-  return generateBtcWallet(!!options.testnet, mnem)
+export const generateWallet = (mnemonic: string, options?: { testnet?: boolean }) => {
```

Comment:
mnemonic was optional

Reply:
This is required too according to master branch.